lib/foreman/model.rb

```diff
@@ -1,2 +0,0 @@
-module Foreman::Model
-end
```

Comment:
Hmm this would change indentation of many files, I will put it back with comment:

```
app/models/compute_resources/foreman/model/ovirt.rb
4:module Foreman::Model

app/models/compute_resources/foreman/model/openstack.rb
1:module Foreman::Model

app/models/compute_resources/foreman/model/gce.rb
1:module Foreman::Model

app/models/compute_resources/foreman/model/vmware.rb
4:module Foreman::Model

app/models/compute_resources/foreman/model/ec2.rb
1:module Foreman::Model

app/models/compute_resources/foreman/model/libvirt.rb
1:module Foreman::Model

app/models/compute_resources/foreman/model/rackspace.rb
1:module Foreman::Model
```